charts/orchestrator/templates/rhdh-operator.yaml

```diff
@@ -325,6 +362,8 @@ data:
           target: https://github.com/parodos-dev/workflow-software-templates/blob/main/entities/workflow-resources.yaml
```

Comment:
shouldn't it be a raw link? Like this one?
https://raw.githubusercontent.com/parodos-dev/workflow-software-templates/main/entities/workflow-resources.yaml